weltyt.py

```diff
@@ -122,7 +122,16 @@ def buy_lands(lands):
     },
 }
 
+commands = {"Help":"Explain all commands", "show_my_resources": "Shows you what resources you have exactly in this time.", "buy": "shows what is possible to buy in this round.", "Plants": "Opens posibility of buying plants.", "Lands": "Opens posibility of buying lands."}
```

Comment:
I was rather thinking that it should be something like this:
`commands = {"buy": "Buy", "show_my_resources": "Show my resources", "help": "Help"}`
In that way you can change the later asserts to something like this:
`if first_choice == commands["show_my_resources"]:`

Thanks to that we can separate "machine" readable commands (like `show_my_resources`) from more human-readable form. We could change the user command anytime using only `commands` dictionary without the need to change it in the logic itself.

Take a simple example. We want to change `Help` command to `What the fuck is happening`.
All we have to do is to change this item:
`{ "help": "Help"}`
to:
`{"help": "What the fuck is happening"}`
and the assert will still stay the same:
`if first_choice == "help":`